tests/udf/test_stddev.py

```diff
@@ -23,14 +23,14 @@ def test_stddev(lt_ctx, use_roi):
         Context class for loading dataset and creating jobs on them
     """
     data = _mk_random(size=(30, 3, 516), dtype="float32")
-    # FIXME the tiling in signal dimension can only be tested once MemoryDataSet
-    # actually supports it
-    dataset = MemoryDataSet(data=data, tileshape=(3, 2, 257),
-                            num_partitions=2, sig_dims=2)
     if use_roi:
+        dataset = MemoryDataSet(data=data, tileshape=(3, 2, 516),
+                                num_partitions=2, sig_dims=2)
         roi = np.random.choice([True, False], size=dataset.shape.nav)
         res = run_stddev(lt_ctx, dataset, roi=roi)
     else:
+        dataset = MemoryDataSet(data=data, tileshape=(3, 2, 257),
+                                num_partitions=2, sig_dims=2)
```

Comment:
@uellue Is this OK? Maybe additionally test w/ contiguous tileshape and no `roi`?

Reply:
Looks perfect, if it passes. Maybe with a FIXME to change it back as soon as ROIs are supported? Do we have an issue or item for that already?

Reply:
Updated `MemoryDataSet` with very simple support for roi w/ arbitrary tileshapes, so we won't need the FIXME at all.